beetsplug/mbsync.py

```diff
@@ -27,10 +26,95 @@
 MBID_REGEX = r"(\d|\w){8}-(\d|\w){4}-(\d|\w){4}-(\d|\w){4}-(\d|\w){12}"
 
 
+def track_performers(info):
```

Comment:
It would be really great to have detailed docstrings for this and `album_performers`. These are pretty complicated functions, so it's not exactly obvious what role they play.

Reply:
Done, is that enough?